src/test/java/edu/cornell/kfs/concur/batch/fixture/PdpFeedFileBaseEntryFixture.java

```diff
@@ -12,92 +15,239 @@
 import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedHeaderEntry;
 import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedPayeeIdEntry;
 import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedTrailerEntry;
+import edu.cornell.kfs.concur.ConcurTestConstants.PdpFeedFileConstants;
 
-public class PdpFeedFileBaseEntryFixture {
-    public static PdpFeedFileBaseEntry buildPdpFile() {
-        PdpFeedFileBaseEntry pdpFile = new PdpFeedFileBaseEntry();
-        pdpFile.setHeader(buildHeader());
-        pdpFile.setVersion("1.0");
-        pdpFile.setTrailer(buildTrailer());
-        pdpFile.setGroup(buildGroups());
-        return pdpFile;
-    }
-
-    private static PdpFeedHeaderEntry buildHeader() {
-        PdpFeedHeaderEntry header = new PdpFeedHeaderEntry();
-        header.setChart("IT");
-        header.setCreationDate("02/17/2017");
-        header.setSubUnit("CLIF");
-        header.setUnit("CRNL");
-        return header;
-    }
-
-    private static PdpFeedTrailerEntry buildTrailer() {
-        PdpFeedTrailerEntry trailer = new PdpFeedTrailerEntry();
-        trailer.setDetailCount(new Integer(1));
-        trailer.setDetailTotAmt(new KualiDecimal(96.95));
-        return trailer;
-    }
+public enum PdpFeedFileBaseEntryFixture {
 
-    private static List<PdpFeedGroupEntry> buildGroups() {
-        List<PdpFeedGroupEntry> groups = new ArrayList();
-        groups.add(buildGroup("Bimbo Foods Inc"));
-        return groups;
+    MARSHAL_TEST(HeaderEntry.BASIC_HEADER, TrailerEntry.BASIC_TRAILER, EnumSet.of(GroupEntry.MARSHAL_TEST)),
```

Comment:
To simplify the setup of these enums, you could have the last constructor arg be a var-arg and then pass it into EnumSet.of() to create the Set.